### PR TITLE
quick steps to contribution to this project

### PR DESCRIPTION
**Pull request by githubstudent**
This is in relation to #1

### CODE REVIEW

**Comment by githubstudent** [comment]
@matthewmccullough and @githubteacher: can you review this?

**Comment by githubstudent** [comment]
@jaw6, you too!

**Comment by jaw6** [comment]
Looks good to me :+1:

**Comment by githubteacher** [comment]
This looks good! 👍 :shipit:

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # My First Project
 This is a practice repository for learning about  GitHub in the Web Interface.
+
+## How to Contribute
+Steps for contributing can be found in the [CONTRIBUTING.md](/CONTRIBUTING.md) file.
```
Comment (githubstudent) [inline README.md:5]: Nice use of relative links if i do say so myself!

**Pull request merged**